penny_u_django/home/static/home/style.css

```diff
@@ -0,0 +1,38 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto+Slab:700&display=swap');
```

Comment:
I think we should add a font for our paragraphs as well. I like Lato. I also wonder if we should use Regular Roboto Slab instead of Bold, it seems a bit heavy.

`@import url('https://fonts.googleapis.com/css?family=Lato|Roboto+Slab&display=swap');`